os/linux/archlinux.go

```diff
@@ -0,0 +1,30 @@
+package linux
+
+import (
+	"strings"
+
+	"github.com/k0sproject/rig"
+	"github.com/k0sproject/rig/os"
+	"github.com/k0sproject/rig/os/registry"
+)
+
+// ArchlinuxARM provides OS support for ArchlinuxARM systems
+type ArchlinuxARM struct {
+	os.Linux
+}
+
+func init() {
+	registry.RegisterOSModule(
+		func(os rig.OSVersion) bool {
+			return os.IDLike == "arch"
+		},
+		func() interface{} {
+			return ArchlinuxARM{}
+		},
+	)
+}
+
+// InstallPackage installs packages via pacman
+func (c ArchlinuxARM) InstallPackage(h os.Host, s ...string) error {
+	return h.Execf("pacman -S --noconfirm %s", strings.Join(s, " "))
```

Comment:
Needs `sudo`, maybe add `--noprogressbar` to get a bit cleaner log output.

```suggestion
	return h.Execf("sudo pacman -S --noconfirm --noprogressbar %s", strings.Join(s, " "))
```